src/tools/generate/generate_chapters.js

```diff
@@ -72,10 +125,67 @@ const write_template = async (language, year, chapter, metadata, body, toc) => {
   });
 
   await fs.outputFile(path, fomatted_html, 'utf8');
-
   await size_of(path);
 };
 
+const write_sitemap_start = async (language, year) => {
+  
+  await fs.outputFile(sitemap_path, '<urlset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9 http://www.sitemaps.org/schemas/sitemap/0.9/sitemap.xsd">\n', 'utf8');
```

Comment:
@rviscomi It's xml and not html, but would it make sense to use the an ejs template for this - like we do for the chapter/ toc itself?

Reply:
I actually implemented that! But that means it would be dynamically generated? So I ripped it out as we would need to iterate through each chapter to get the last_updated timestamp, so that would be slow. Unless we moved that date into config? But as per our other converstion I quite like it where it is as can be different per chapter and language...

To me it is simple enough and static so not sure there is need for ejs template.

Reply:
It would be static. We're just using ejs to help with the code generation.